lib/ansible/modules/packaging/os/dnf.py

```diff
@@ -616,6 +617,26 @@ def _base(self, conf_file, disable_gpg_check, disablerepo, enablerepo, installro
                 results=[],
                 rc=1
             )
+        except AttributeError as e:
+            if 'demands' in to_text(e):
+                version_lock_index = None
+                for plugin in base._plugins.plugins:
+                    if plugin.name == "versionlock":
+                        self.module.warn("Disabling dnf versionlock plugin due do known upstream bug: "
+                                         "https://github.com/rpm-software-management/dnf-plugins-core/pull/317")
+                        self.disable_plugin.append('versionlock')
+                        broken_plugin_found = True
```

Comment:
Where is `broken_plugin_found` used?

Reply:
It's not, I forgot to remove that. I was toying with a different implementation but this ended up being less invasive.